app/pages/patients/patients.js

```diff
@@ -98,7 +94,7 @@ export let Patients = React.createClass({
     var handleClickYes = function(e) {
       e.preventDefault();
       self.props.onHideWelcomeSetup();
```

Comment:
I don't think we want to trigger `onHideWelcomeSetup` on clicking yes, right? Because if I get distracted and close the tab, then come back I won't get the welcome setup. OTOH, if I complete the welcome setup, my patients will now be > 0 so it won't matter that I have `showingWelcomeMessage` true still.